listenbrainz/webserver/static/css/theme/boostrap/pager.less

```diff
@@ -46,7 +46,7 @@
     > a:hover,
     > a:focus,
     > span {
-      color: @pager-disabled-color;
+      color: @pager-disabled-color !important;
```

Comment:
This won't be needed in the end, and `!important` rules in CSS make me feel a bit uncomfortable :)
I see the issue you were solving with this, and found the root cause of it that we can fix: the `<a>` tags in our navigation have an empty `href` attribute, which causes the `.disabled` style not to be applied.

We recently corrected similar links in #796 so what I suggest is changing those anchor with empty `href` in UserEntityChart.tsx into:

```
<a
  href={`?page=${Math.max(prevPage,1)}&range=${range}&entity=${entity}`}
  role="button"
  onClick={(event) => this.changePage(prevPage, event)}
>
```
                      
and
```
<a
  href={`?page=${Math.min(nextPage,totalPages)}&range=${range}&entity=${entity}`}
  role="button"
  onClick={(event) => this.changePage(nextPage, event)}
>
```

This will fix the css styling issue and as a bonus also allows users to right-click on the navigation buttons and open in a new tab, for example.